### PR TITLE
Hemangsk/css

### PR DESCRIPTION
**Pull request by hemangsk**
Short Note (same as landing-frontend)

This has a lot of changes in all the partials file. These changes are large in number, but essentially they do only these four tasks,

- Removes Bootstrap's class and use MaterializeCSS grid classes.
- Uses classes available in coalaCSS and restructures the divs accordingly if required.
- Indentation and removal of extra newlines.
- Lastly, Introduces the usage of coalaCSS via CDN link from rawgit. Its a CSS theme which we can use throughout web projects at coala.

It will be really awesome if someone can help me test the changes! So to fetch these new changes and to test locally,

```
git clone https://github.com/coala/projects.git
cd projects
git fetch origin pull/297/head:hemangsk/css
git checkout hemangsk/css
python -m SimpleHTTPServer 8080 # python2
# or
python3 -m http.server 8080 #python3
```
Visit localhost:8080 in browser to view the site running.
Thanks ☺️

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by sils** [comment]
@hemangsk just put an ignore there and file an issue for the bear?

File: partials/tabs/mentors.html [hunk 1]
```diff
@@ -1,59 +1,68 @@
...
+                        <div class="empty"></div>
+                        <div class="data thumbnail center" >
+                          <img class="center dp" height="100" ng-src="{{ member.github_avatar_url }}">
+                          <div class="name center">{{ member.name }}</div>
+                          <div class="handle center"><a class="yellow-text text-darken-4" ng-href="https://github.com/{{member.github_handle}}">@{{ member.github_handle }}</a></div>
+                        </div>
+                    </div>
                 </div>
-            </div>
+            </div>
```
Comment (sils) [inline partials/tabs/mentors.html]: trailing whitespace in https://github.com/coala/projects/pull/297/commits/974c030aaa6423cf83dd866c289e61d2897455a4

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/297/commits/b1336d2f4291952a0408873181c2c33f3cd01431 https://github.com/coala/projects/pull/297/commits/43b8a2ba14066d44726376c7e4b4c69154fd02c5 https://github.com/coala/projects/pull/297/commits/70a181940d438611c6b898893fc5e1dc5ca6ee64 https://github.com/coala/projects/pull/297/commits/a3d628274913ff8637e694dd6f2ed3538f4a3995 https://github.com/coala/projects/pull/297/commits/f09a5a7c80351c6ad62e033e71962bd2f4a95470 https://github.com/coala/projects/pull/297/commits/39e2a67f425bb045c69448ec2b6137fb0dc1b28f

**Comment by hemangsk** [comment]
reack 7292161 ca045ca

**Comment by hemangsk** [comment]
reack 7292161

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/297/commits/ae888344d24ab092151c90baf52d666bbd98ecfd

**Pull request merged**